### ISSUE TITLE
Minor battle bugs in OFF

### ISSUE THREAD

**Issue by rsclifford** [EasyRPG/Player#1277]
Please fill in the following fields before submitting an issue:
#### Name of the game:
OFF (v1.6.1, English 2.0 Update)

#### Player platform:
Windows

#### Attach files (as a .zip archive or link them)
[easyrpg_log.txt](https://github.com/EasyRPG/Player/files/1366434/easyrpg_log.txt)
[Save01.zip](https://github.com/EasyRPG/Player/files/1366433/Save01.zip)

#### Describe the issue in detail and how to reproduce it:
Two bugs I've run into with battles in OFF:

- [x] The "Purification in progress" message at the beginning of battles doesn't appear #2364
- [x] The characters don't animate when using physical attacks: ex. when Alpha uses a physical attack, it should rush at the targeted enemy, collide, and fly back into position. (since 0.6.3, #2503)

To reproduce:
- Load the attached save
- Walk around until you get into a battle
- Use a physical attack

**Comment by carstene1ns** [comment]
Again, thanks for the bug report.
Did a quick search for the term `battle_start`. It is currently not used, so this is not implemented yet.
We only have a part of the possible animations already, some are missing. Did not check what is used here, but high chance this is also not implemented yet.

**Comment by Ghabry** [comment]
Rush and collide could be #1269

**Comment by carstene1ns** [comment]
Patch for the battle start message:

```patch
diff --git a/src/scene_battle_rpg2k3.cpp b/src/scene_battle_rpg2k3.cpp
index e301cb9a..1a0304fc 100644
--- a/src/scene_battle_rpg2k3.cpp
+++ b/src/scene_battle_rpg2k3.cpp
@@ -394,6 +394,11 @@ void Scene_Battle_Rpg2k3::SetState(Scene_Battle::State new_state) {
 
 	switch (state) {
 	case State_Start:
+		if (!Data::terms.battle_start.empty()) {
+			help_window->SetVisible(true);
+			message_timer = 120;
+			help_window->SetText(Data::terms.battle_start);
+		}
 		break;
 	case State_SelectOption:
 		options_window->SetVisible(true);
```
Problem is, that the message is not displayed long enough... :/

**Comment by Ghabry** [comment]
You can do it in the switch-case in ProcessActions, there is the state transition to State_SelectOption.

**Comment by carstene1ns** [comment]
I had it there before, however, the window was not shown at all then.

**Comment by fmatthew5876** [comment]
The first issue was likely fixed by #2364 (needs a test to verify)

The second one sounds like duplicate of CBA

**Comment by Ghabry** [comment]
Yeah Purification is the start message
